bin/ruskquery

```diff
@@ -34,23 +34,16 @@ query_block_height() {
 }
 
 query_connected_peers() {
-    curl -s --location --request POST "$API_ENDPOINT" \
+    curl -s --location --request POST "$API_ENDPOINT/on/network/peers" \
     --header "Rusk-Version: $RUSK_VERSION" \
     --header "Content-Type: $CONTENT_TYPE" \
-    --data-raw '{
-        "topic": "alive_nodes",
-        "data": " 1000"
-    }' | jq '. | length'
+    --data-raw '100' | jq '. | length'
```

Comment:
Please double check the 100 peers assumption and introduce a suitable value

Reply:
Due to the way the called endpoint works, it will only retrieve up to the value passed in raw. Since the purpose of this functionality is mainly to see if there's any connected peers at all, I've opted to display "+99" if there's more than 99 peers connected. The actual peer IPs are not used this script